src/test/shared/sam/cli/samCliInit.test.ts

```diff
@@ -195,4 +204,44 @@ describe('runSamCliInit', async () => {
 
         await runSamCliInit(sampleSamInitArgs, context)
     })
+
+    describe('runSamCliInit With EventBridgeStartAppTemplate', async () => {
```

Comment:
You have a "runSamCliInit With EventBridgeStartAppTemplate" scope within a "runSamCliInit with HelloWorld template" which is awkward.

Suggest restoring the "runSamCliInit" scope and placing both of those other scopes within "runSamCliInit"

Reply:
Sounds good, restored back the original "runSamCliInit" and have two separate scopes inside it. One for  HelloWorld template and the other  EventBridgeStartAppTemplate